### DISCUSSION TITLE
plan line

### DISCUSSION THREAD

**Discussion by AbaElMonzer** [flybywiresim/aircraft#9427] [Help]
![Screenshot 2024-09-20 151745](https://github.com/user-attachments/assets/d986419a-4a41-4e4e-8438-15b2e0b0a857)
![Screenshot 2024-09-20 151815](https://github.com/user-attachments/assets/8a19ca99-67ef-4162-a16e-944f6b82dca7)
I don't find line in plan desplay

**Comment by tracernz** [comment]
Remove any mods like the WxR mod from flightsim.to. Unfortunately people keep doing mods that conflict with our aircraft.

**Comment by AbaElMonzer** [comment]
Thank you very muchThe problem has been solved

    On Wednesday, November 6, 2024 at 09:41:52 PM GMT+2, Michael Corcoran ***@***.***> wrote:  
 
 


Remove any mods like the WxR mod from flightsim.to. Unfortunately people keep doing mods that conflict with our aircraft.
